external/fv3fit/fv3fit/train.py

```diff
@@ -42,12 +42,6 @@ def get_parser():
             "by default an empty sequence is used"
         ),
     )
-    parser.add_argument(
-        "--local-download-path",
-        type=str,
-        default=None,
-        help=("path for downloading data before training"),
-    )
```

Comment:
Make sure people are aware of the change they need to make to `run.sh` or similar because of this.